lib/republish_content.rb

```diff
@@ -1,7 +1,7 @@
 class RepublishContent
   def self.schedule_republishing
     Edition.published.each do |edition|
-      PublishingAPINotifier.perform_async(edition.id.to_s, "republish")
+      PublishingApiPublisher.perform_async(edition.id.to_s, "republish")
```

Comment:
This no longer really does what is expected by "republish" - that is, re-send the content to publishing-api. We need to call both put_content and publish; I think unfortunately this means we will need a third worker class, PublishingAPIRepublisher, which does both actions, as calling both existing workers individually will lead to out-of-order problems.